.travis.yml

```diff
@@ -4,7 +4,5 @@ node_js:
   - "10"
   - "8"
   - "6"
-  - "4"
```

Comment:
TBH this version is fine, it's just har-validator seems to prefer 6+. I didn't notice this when I was testing locally because I didn't reinstall dev deps between versions.